src/software/gui/full_system/threaded_full_system_gui.cpp

```diff
@@ -46,8 +46,14 @@ ThreadedFullSystemGUI::~ThreadedFullSystemGUI()
     run_full_system_gui_thread.join();
 }
 
-void ThreadedFullSystemGUI::createAndRunFullSystemGUI(int argc, char** argv)
+void ThreadedFullSystemGUI::createAndRunFullSystemGUI()
 {
+    // We mock empty argc and argv since they don't affect the behaviour of the GUI.
+    // This way we don't need to pass them all the way down from the start of the
+    // program
+    char* argv[] = {NULL};
+    int argc     = sizeof(argv) / sizeof(char*) - 1;
```

Comment:
are these being used anywhere anymore?

Reply:
Yes, used to create the `QApplication` in this thread.